src/GitVersion.Core/Configuration/IIgnoreConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+using GitVersion.VersionCalculation;
+
+namespace GitVersion.Configuration
+{
+    public interface IIgnoreConfiguration
+    {
+        DateTimeOffset? Before { get; }
+
+        IReadOnlyList<string> Shas { get; }
+
+        IEnumerable<IVersionFilter> ToFilters();
```

Comment:
Question, can we move this method out of the interface/record, to an extension maybe or private method at the place of usage, Currently there are only 2 usages.

Reply:
Could be done in the other PR

Reply:
Yep good idea. :) done